plugins/sbt/graphql-java-codegen-sbt-plugin/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.4.7
\ No newline at end of file
+sbt.version=1.4.7
```

Comment:
All Scala files should be ignored, and sbt example does not need to be formatted.

Reply:
Ok. I will do it tomorrow

Reply:
Actually, what's wrong with this particular change? Checkstyle did not touch scala files. Just `.properties` files

Reply:
Does not conform to the scala project common format. But it's no problem.